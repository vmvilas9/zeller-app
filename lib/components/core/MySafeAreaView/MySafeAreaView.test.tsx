import renderer from 'react-test-renderer';
import {MySafeAreaView} from "./index";
import * as React from "react";

it(`renders correctly`, () => {
    const tree = renderer.create(<MySafeAreaView>Test</MySafeAreaView>).toJSON();

    expect(tree).toMatchSnapshot();
});
