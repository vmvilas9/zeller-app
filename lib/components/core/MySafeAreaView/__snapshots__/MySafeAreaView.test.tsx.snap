// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RNCSafeAreaView
  edges={
    {
      "bottom": "additive",
      "left": "additive",
      "right": "additive",
      "top": "additive",
    }
  }
  style={
    {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "padding": 30,
    }
  }
>
  Test
</RNCSafeAreaView>
`;
